doc/classes/EditorPlugin.xml

```diff
@@ -675,6 +682,12 @@
 				Removes an inspector plugin registered by [method add_import_plugin]
 			</description>
 		</method>
+		<method name="remove_move_array_element_function">
+			<return type="void" />
+			<param index="0" name="class" type="String" />
+			<description>
+			</description>
+		</method>
```

Comment:
Please add some docs here too, although, this could be very simple like:

```
Removes a move array element function registered by [method add_move_array_element_function].
```